src/app/(proper_react)/(redesign)/(public)/FreeScanCta.tsx

```diff
@@ -88,12 +82,13 @@ export const FreeScanCta = (
   ) : (
     <div>
       <TelemetryButton
+        buttonRef={refViewTelemetry as unknown as RefObject<HTMLButtonElement>}
```

Comment:
I wonder if it maybe would make sense to just add this type assertion (or I suppose `RefObject<any>`) to the return value of `useViewTelemetry`?

Reply:
Or maybe, clicking through to the type definition of the `ref` prop on `<button>` and the type definition of react-intersection-observer, `RefCallback` would be even more accurate, but in practice, I don't think it matters in practice.

Reply:
I moved the type assertion to the `useTelemetryView` view hook in [6618a55](https://github.com/mozilla/blurts-server/pull/4794/commits/6618a558dc6acc841886a6187cc795998b4f7eae). I’ve kept `RefObject`, because otherwise we would need to convert back to `unknown` again when using the return value of the hook.